### DISCUSSION TITLE
Langflow JSON to Python Code Conversion

### DISCUSSION THREAD

**Discussion by okulkarni-eis** [langflow-ai/langflow#325] [General]
Hello,

I am reaching out to inquire if there is a possibility to convert the 'langflow' JSON into executable Python code. It is crucial for us to incorporate logging statements and invoke custom code within the implementation. Any assistance or guidance you can provide in achieving this would be greatly appreciated.

Thank you in advance for your support.

**Comment by Cristhianzl** [comment]
hi @okulkarni-eis 

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
